website/gatsby-browser.js

```diff
@@ -17,37 +17,34 @@
 
 exports.onClientEntry = () => {
   // Check if the URL on load has a hash, and navigate to the position on page
-  if (window && window.document && window.location && window.location.hash) {
-    const { location } = window;
-    const el = window.document.getElementById(location.hash.replace('#', ''));
-
-    if (el) {
-      const rect = el.getBoundingClientRect();
-      window.scrollTo(0, rect.top);
-    }
-  }
+  // if (window && window.document && window.location && window.location.hash) {
+  //   const { location } = window;
+  //   const el = window.document.getElementById(location.hash.replace('#', ''));
+  //   if (el) {
+  //     const rect = el.getBoundingClientRect();
+  //     window.scrollTo(0, rect.top);
+  //   }
+  // }
```

Comment:
This actually fixes an existing bug unrelated to this PR that I've noticed for a while, that the hash navigation doesn't work. Or did not work but it works now on this preview. What happened there and why does this fix it? Just curious

Reply:
This is a default behavior of the browser when using hash navigation. With the current code I couldn't  use the hash .
So I just commented this code to try to fix it  it and it started working